### ISSUE TITLE
Including an Invalid Relation Shouldn't Throw Exception

### ISSUE THREAD

**Issue by brunogaspar** [flugger/laravel-responder#73]
Hi

First, thanks for the package, it has been really helpful :)

Now for the issue.

If i have defined a list of available relations on the transformer and then when i do a request and i request to include a relation that's not on that list and at the same time is invalid (doesn't exist on the model), it throws an exception about the relation being invalid, IMO, that shouldn't throw the exception, at all, since that's not on the list of the available relations.

Transformer:

```php
<?php

namespace App\Transformers;

use App\Models\Client;

class ClientTransformer extends Transformer
{
    /**
     * List of available relations.
     *
     * @var array
     */
    protected $relations = ['invoices'];

    // Truncated for brevity
}
```

If i hit `http://foo.dev/clients?with=invalid_relation` an exception is thrown.

And if for example, i make that array empty and require `?with=invoices` it doesn't throw an error, which is kinda expected, since it's not allowed.

I know i can catch the exception and deal with it, however, i don't feel that this is the correct behaviour.

Thanks!

**Comment by flugger** [comment]
Hi @brunogaspar, I agree with your sentiment! I'll look into it soon, hopefully tonight. Thanks!

**Comment by brunogaspar** [comment]
Thank you @flugger !

**Comment by flugger** [comment]
Sorry for the delay, been a hectic week at work. I'll try to look at this over the weekend

**Comment by brunogaspar** [comment]
No worries, take your time.

Thanks!

**Comment by flugger** [comment]
I feel bad about this taking so long, but at least this should be fixed with the latest `v3.0.0` release! Let me know if you still have problems
